packages/shared/src/components/CombatReport/CombatVideo/index.tsx

```diff
@@ -141,6 +139,24 @@ export const CombatVideo = () => {
     return null;
   }
 
+  let timeBasis = 0;
+  // Video describes an activity that started normally at an offset inside a buffer
+  // the timebasis will now just be the number of seconds needed to adjust for the keyframe cut from ffprobe
+  if (videoInformation.compensationTimeSeconds > 0) {
+    timeBasis = videoInformation.compensationTimeSeconds;
+  }
+  // Video describes an activity that started before the video's buffer began
+  // the timebasis will now be negative to account for the missing video footage and synchronize the timeline data
+  // All events before the video start will be imputed to t=0
+  if (videoInformation.relativeStart < 0) {
```

Comment:
we can't store a negative value in compenstationTimeSeconds field?

Reply:
we could, it just represents a different calculation and I want to be able to debug them separately (they're both going in the analytics event as well)